paraview/CMakeLists.txt

```diff
@@ -46,11 +46,15 @@ paraview_plugin_build(
     "${CMAKE_INSTALL_BINDIR}/plugins"
   LIBRARY_DESTINATION
     "${CMAKE_INSTALL_BINDIR}/plugins"
+  PLUGINS_FILE_NAME
+    "ttk.plugins.xml"
```

Comment:
Does that file get created at runtime?
It doesn't look like it's in the source tree right now.